pkg/gcs/gcs.go

```diff
@@ -22,6 +22,30 @@ import (
 	"google.golang.org/api/iterator"
 )
 
+func UploadFile(ctx context.Context, srcFile io.Reader, destURL string, publish bool) error {
+	destURL = strings.TrimPrefix(destURL, "gs://")
+	client, err := NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("func NewClient: %w", err)
```

Comment:
Nit: Can you make the log messages more meaningful?

Reply:
My logic is:
1. Extending some error message, the only information is required is the callstack. Almost every fmt.Errorf() with %w parameter is a candidate for this approach.
2. Creating the error message, it makes sense to give more detailed description. fmt.Errorf w/o parameters or errors.New are the cases for it.